### ISSUE TITLE
Track doesn't align in RN0.39

### ISSUE THREAD

**Issue by Thomas101** [jeanregisser/react-native-slider#54]
Hi, I've noticed the track doesn't align properly in React Native 0.39.

```js
<Slider
  style={{width: 300}}
  value={this.state.value}
  minimumTrackTintColor='#FF0000'
  maximumTrackTintColor='#00FF00'
  thumbTintColor='#0000FF'
  minimumValue={0}
  maximumValue={10}
  onValueChange={(value) => this.setState({value})} />
```

As a short term fix, I've found that providing  `trackStyle={{marginTop: -4}}` fixes the issue, but I suspect changing `defaultStyles` to include the fix would be a better solution.

Here's a screenshot of the bug...
![screen shot 2016-12-05 at 15 46 06](https://cloud.githubusercontent.com/assets/103586/20891346/2daf8c3c-bb02-11e6-9dfc-1a43745bec1f.png)

**Comment by 3610cn** [comment]
i meet same problem

**Comment by jsdario** [comment]
Trying to solve this, any suggestions?

**Comment by Thomas101** [comment]
@3610cn did adding `trackStyle={{marginTop: -4}}` fix it for you?

**Comment by jsdario** [comment]
Worked for me, thanks @Thomas101

**Comment by mikaello** [comment]
More specifically: `trackstyle={{ margin: -trackHeight }}` (where trackHeight is the height of your track) worked for me, I guess 4 is the default height of the track.

**Comment by UI-Animation-Chen** [comment]
thanks a lot :)

**Comment by renso3x** [comment]
thanks @Thomas101

**Comment by jeanregisser** [comment]
A proper fix for this has (finally) been landed in the latest published version (0.10.0)!
Thanks everyone.
